### ISSUE TITLE
[v13.0.0-beta.2] Toast and aria-live text punctuation

### ISSUE THREAD

**Issue by microbit-robert** [google/blockly#9801]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Description

The aria-live region now coalesces messages together when required, however, the messages lack punctuation, and when they are concatenated, the screen outputs one long sentence with varying levels of sense. 

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

### Screenshots

Example: "moving on workspace" is appended to the move mode hint but no full stop separates these two distinct sentences so they are no longer distinct.
<img width="662" height="196" alt="Image" src="https://github.com/user-attachments/assets/a3be8b5d-0075-4c51-a814-ced9b0cc7d7a" />

**Comment by maribethb** [comment]
Aaron implemented with newlines but that doesn't appear to be working with voiceover

options:
- make sure the messages themselves have proper punctuation so they're already separated naturally
- add additional punctuation like `/` or `-` or something
- both?

there's no drawback to doing #1 and I think also doing the second is fine as well

**Comment by mikeharv** [comment]
This PR addresses option 1 from @maribethb in the comment above: https://github.com/RaspberryPiFoundation/blockly/pull/9817

If this sufficiently addresses the problem, we should be able to close the ticket on merge. Otherwise we can consider option 2 (additional punctuation).
